2002-05-08  Jason Merrill  <jason@redhat.com>

	* g++.dg/debug/const1.C: New test.

--- gcc/testsuite/g++.dg/debug/const1.C.jj	Thu Aug 30 22:30:55 2001
+++ gcc/testsuite/g++.dg/debug/const1.C	Wed May  8 17:33:04 2002
@@ -0,0 +1,11 @@
+// PR c++/6381
+// Bug: we were emitting the initializer for bar, which referenced foo,
+// which was not emitted.
+
+// { dg-options "-O" }
+// { dg-do link }
+
+static const int foo[] = { 0 };
+static const int * const bar[] = { foo };
+
+int main() {}
